pkg/generate/app/componentresolvers.go

```diff
@@ -197,21 +154,18 @@ type HighestUniqueScoreResolver struct {
 // Resolve resolves as the highest scored match returned by the Searcher, and
 // guarantees the match is unique (the only match with that given score)
 func (r HighestUniqueScoreResolver) Resolve(value string) (*ComponentMatch, error) {
```

Comment:
do we really have use cases for all these resolvers?  (first, highest, highest unique)
seems like "highest score" by itself ought to be sufficient.